app/channels/application_cable/channel.rb

```diff
@@ -1,4 +0,0 @@
-module ApplicationCable
```

Comment:
why did you remove the files? don't you need this?

Reply:
We did this to increase test coverage, this files are not used in the application